### ISSUE TITLE
No warning shown for space-before-brace rule

### ISSUE THREAD

**Issue by alurim** [skovhus/SublimeLinter-contrib-sass-lint#5]
My .sass-lint.yml file doesn't change the space-before-brace rule and when I run `sass-lint -v <file>` it shows that there is a problem with no space shown

In Sublime:
![screen shot 2015-11-11 at 10 01 38 am](https://cloud.githubusercontent.com/assets/15201138/11098653/4592fe0e-885b-11e5-9123-c80a7780d028.png)

Running `sass-lint` in the command line:
![screen shot 2015-11-11 at 10 03 16 am](https://cloud.githubusercontent.com/assets/15201138/11098681/7196d99e-885b-11e5-87a9-6e314319ba43.png)

Am I missing something or is this a valid issue?

**Comment by skovhus** [comment]
@alurim thanks for reporting this. I cannot reproduce this with `sass-lint@1.3.3`. Which version are you running? Can I get you to open the console in Sublime by `View -> Show Console` and dump what is in there. For me I get:

```
  1:6  warning  Whitespace required before {           space-before-brace
  2:5  warning  Expected `line-height`, found `width`  property-sort-order
  3:5  warning  Expected `width`, found `line-height`  property-sort-order
```

And it shows up:
<img width="248" alt="screenshot 2015-11-27 00 04 12" src="https://cloud.githubusercontent.com/assets/1260305/11431536/7fb00d7a-949a-11e5-925e-250c38f02e53.png">

**Comment by gjjoo** [comment]
I like this.
My version is 1.4.0

**Comment by skovhus** [comment]
@gjjoo @alurim is this still valid? Else I'm closing this issue.

**Comment by alurim** [comment]
Looks good to me now, thanks!

**Comment by skovhus** [comment]
@alurim cool, I guess it was an issue in `sass-lint` : )
